src/Infrastructure/Persistence/Configuration/TrustLevelConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Codidact.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Codidact.Infrastructure.Persistence.Configuration
+{
+    /// <summary>
+    /// Mapping configuration for the Member entity
+    /// </summary>
+    public class TrustLevelConfiguration : IEntityTypeConfiguration<TrustLevel>
+    {
+        public void Configure(EntityTypeBuilder<TrustLevel> builder)
+        {
+            builder.Property(t => t.Content)
```

Comment:
This (and L18) will need changing to match the update to `LevelName`.

Reply:
Fixed in 43dd079f0332e6f56dbf41724fbb40e3f704c7bc